yb-voyager/cmd/importDataState.go

```diff
@@ -796,7 +806,25 @@ type Batch struct {
 }
 
 func (batch *Batch) Open() (*os.File, error) {
-	return os.Open(batch.FilePath)
+	file, err := os.Open(batch.FilePath)
+	if err != nil {
+		return nil, err
+	}
+	return file, err
+}
+
+func (batch *Batch) OpenDataFile() (datafile.DataFile, error) {
```

Comment:
this is for reading the batch file right? 
Rename function to OpenBatchAsDataFile()